File: LATIHAN2/Latihan2_Python/Hardware.py
from statistics import mode
from Product import Product


class Hardware(Product):
    # class attribute
    __model = ""
    __brand = ""

    # constructors
    def __init__(self, model, brand):
        self.__model = model
        self.__brand = brand

    # getter and setter
    def setModel(self, model):
        self.__model = model

    def getModel(self):
        return self.__model

    def setBrand(self, brand):
        self.__brand = brand

    def getBrand(self):
        return self.__brand

File: LATIHAN2/Latihan2_CPP/Hardware.cpp
#include <string>
#include "Product.cpp"

using namespace std;

class Hardware : public Product
{
private:
	/* data */
	string model;
	string brand;

public:
	Hardware(){

	};

	Hardware(string model, string brand)
	{
		this->model = model;
		this->brand = brand;
	};

	// getter and setter
	void setModel(string model)
	{
		this->model = model;
	};

	string getModel()
	{
		return this->model;
	};

	void setBrand(string brand)
	{
		this->brand = brand;
	};

	string getBrand()
	{
		return this->brand;
	};
};
